src/PKSim.Core/Model/IndividualProteinTask.cs

```diff
@@ -0,0 +1,273 @@
+      {
+         var globalContainer = CreateMolecule(moleculeName);
+         AddVascularSystemExpressionNew(globalContainer, CoreConstants.Groups.VASCULAR_SYSTEM,
+            relExpParam(REL_EXP_BLOOD_CELLS),
+            fractionParam(FRACTION_EXPRESSED_BLOOD_CELLS, CoreConstants.Rate.ZERO_RATE),
+            fractionParam(FRACTION_EXPRESSED_BLOOD_CELLS_MEMBRANE, CoreConstants.Rate.PARAM_F_EXP_BC_MEMBRANE, editable: false)
+         );
+
+         AddVascularSystemExpressionNew(globalContainer, CoreConstants.Groups.VASCULAR_SYSTEM, relExpParam(REL_EXP_PLASMA));
+
+         AddVascularSystemExpressionNew(globalContainer, CoreConstants.Groups.VASCULAR_SYSTEM,
+            relExpParam(REL_EXP_VASC_ENDO),
+            fractionParam(FRACTION_EXPRESSED_VASC_ENDO_ENDOSOME, CoreConstants.Rate.ZERO_RATE),
+            fractionParam(FRACTION_EXPRESSED_VASC_ENDO_APICAL, CoreConstants.Rate.ZERO_RATE),
+            fractionParam(FRACTION_EXPRESSED_VASC_ENDO_BASOLATERAL, CoreConstants.Rate.PARAM_F_EXP_VASC_BASOLATERAL, editable: false)
+         );
+
+         AddTissueOrgansExpressionNew(simulationSubject, moleculeName);
+         AddLumenExpressionsNew(simulationSubject, moleculeName);
+         AddMucosaExpressionNew(simulationSubject, moleculeName);
+
+         simulationSubject.AddMolecule(globalContainer);
+
+         _individualPathWithRootExpander.AddRootToPathIn(simulationSubject, moleculeName);
+         return globalContainer;
+      }
+
+      protected void AddMucosaExpressionNew(ISimulationSubject simulationSubject, string moleculeName)
+      {
+         foreach (var organ in simulationSubject.Organism.OrgansByName(CoreConstants.Organ.SmallIntestine, CoreConstants.Organ.LargeIntestine))
+         {
+            var organMucosa = organ.Compartment(CoreConstants.Compartment.Mucosa);
+            foreach (var compartment in organMucosa.GetChildren<Compartment>().Where(c => c.Visible))
+            {
+               addTissueParameters(compartment, moleculeName, CoreConstants.Groups.GI_MUCOSA);
+            }
+         }
+      }
+
+      protected void AddLumenExpressionsNew(ISimulationSubject simulationSubject, string moleculeName)
+      {
+         var lumen = simulationSubject.Organism.Organ(CoreConstants.Organ.Lumen);
+         foreach (var segment in lumen.Compartments.Where(c => c.Visible))
+         {
+            AddContainerExpressionNew(segment, segment.Name, moleculeName, CoreConstants.Groups.GI_LUMEN,
+               relExpParam(REL_EXP),
+               initialConcParam(CoreConstants.Rate.INITIAL_CONCENTRATION_LUMEN));
+         }
+      }
+
+      protected virtual MoleculeExpressionContainer AddContainerExpressionNew(IContainer parentContainer, string containerName, string moleculeName,
+         string groupingName,
+         params ParameterMetaData[] parameters)
+      {
+         return addContainerExpressionNew(parentContainer, containerName, moleculeName, groupingName, parameters);
```

Comment:
same

Reply:
same answer. New means that this is the new implementation so that I can compare with the old one